src/actions/exportAction.ts

```diff
@@ -22,17 +24,18 @@ export async function exportAction({
   // eslint-disable-next-line no-warning-comments
   // @TODO use https://listr2.kilic.dev/ to add interactive prompts if spaceId or managementToken are missing
 
-  await contentfulExport({
+  const options = {
     spaceId,
     managementToken,
     environmentId,
     skipContentModel: false,
     skipEditorInterfaces: false,
+    includeDrafts: true,
     // If saveFile is false, then we're not exporting data, we're exporting what we need to generate a schema
     skipWebhooks: true,
     skipRoles: true,
     downloadAssets: false,
-    maxAllowedLimit: 50,
+    maxAllowedLimit: 250,
```

Comment:
I raised this limit to increase the speed of my exports. I think the default is even higher, so should be fine